test_fixtures/rc/transformer_qa/experiment_v2.jsonnet

```diff
@@ -0,0 +1,31 @@
+local transformer_model = "albert-base-v2";
```

Comment:
Didn't you make a special nano-BERT for these cases?

Reply:
Yes, I'll give that a try